src/js/components/DataTable/StyledDataTable.js

```diff
@@ -103,12 +104,11 @@ const StyledDataTableBody = styled(TableBody)`
     overflow: auto;
   `}
 
+  // Needed to give outline styling
   &:focus {
-    ${focusStyle({ skipSvgChildren: true, forceOutline: true })}
-  }
-
-  &:focus:not(:focus-visible) {
-    ${unfocusStyle({ skipSvgChildren: true, forceOutline: true })}
+    ${props =>
+      props.tabIndex >= 0 &&
+      focusStyle({ forceOutline: true, skipSvgChildren: true })}
```

Comment:
Noticing that with mouse click, the focus outline is appearing around the datatable body. We made [recent changes](https://github.com/grommet/grommet/pull/5148) so that focus outline would only appear on keyboard interaction. Is there anyway to have the body receive focus without causing the focus outline on mouse click?

Reply:
Just to clarify, so we would like the DataTable body to only receive the focus outline via keyboard navigation? If so, this should be a quick fix I can look into.